src/main/java/org/jetbrains/bsp/bazel/server/bsp/utils/ParsingUtils.java

```diff
@@ -0,0 +1,25 @@
+package org.jetbrains.bsp.bazel.server.bsp.utils;
+
+import ch.epfl.scala.bsp4j.BuildTargetTag;
+import com.google.common.collect.ImmutableMap;
+import java.util.Map;
+import org.jetbrains.bsp.bazel.commons.Constants;
+
+public class ParsingUtils {
+
+  private static final Map<String, String> RULE_CLASS_TO_RULE_TYPE =
+      ImmutableMap.of(
+          Constants.LIBRARY_RULE_TYPE, BuildTargetTag.LIBRARY,
+          Constants.BINARY_RULE_TYPE, BuildTargetTag.APPLICATION,
+          Constants.TEST_RULE_TYPE, BuildTargetTag.TEST);
+  private static final String DEFAULT_RULE_TYPE = BuildTargetTag.NO_IDE;
+
+  public static String convertOutputToPath(String output, String prefix) {
+    String pathToFile = output.replaceAll("(//|:)", "/");
+    return prefix + pathToFile;
+  }
+
+  public static String getRuleType(String ruleClass) {
+    return RULE_CLASS_TO_RULE_TYPE.getOrDefault(ruleClass, DEFAULT_RULE_TYPE);
```

Comment:
I would consider removing this class altogether and moving the contents of this class to ``BazelBspBuildManager`` (it's only used there) – it too is still pretty strongly related to the logic there, it's not much code and ``BazelBspBuildManager`` should still be split into several pieces in the future.

Reply:
this util is in this package and it is related with manager. Also, in general we do want to split logic, so I believe that it should stay in the separate class

Reply:
+ I rollbacked `getRuleType` method, because I missed `.contains`, what makes inconvenient to use map

Reply:
Sure, just a suggestion. In this case maybe we should at least rename it to something that suggest these are parsing utils for the bsp package and not some general parsing utils? Don't have any good idea though.

Reply:
it is in `.../bsp/utils` do we need specify it more?

Reply:
``BuildParsingUtils`` or ``BuildManagerParsingUtils`` would be fine for now but maybe we'd want to have here something from other places within the bsp package.

Reply:
Actually maybe it is a good idea after all. There are not many such places that make sense.

Reply:
done